src/main/java/com/afornalik/ox/progress/OneMove.java

```diff
@@ -0,0 +1,20 @@
+package com.afornalik.ox.progress;
+
+import com.afornalik.ox.board.*;
+
+public class OneMove {
+
+    private final Board board;
+
+    public OneMove(Board board) {
+        this.board = board;
+    }
+
+    public void makeMove(int indexOfField, FieldStatus fieldStatusX) {
+        try {
+            board.insertBoardField(indexOfField,fieldStatusX);
+        } catch (OutOfBoardException e) {
+            e.printStackTrace();
+        }
```

Comment:
printStackTrace? is this a TODO thing? If it is, mark it as TODO. If not, you can do something better with it (eg get message and show it to user).

Reply:
Replace with custom message from exception.

Reply:
You could use some ui for doing that (just a suggestion)

Reply:
Yes i will replace it soon.